pyresample/test/test_geometry.py

```diff
@@ -31,6 +31,7 @@
 from pyresample import geo_filter, geometry, parse_area_file
 from pyresample.geometry import (
     IncompatibleAreas,
+    SwathDefinition,
```

Comment:
Why is the import added here?  Don't we usually have imports from the module to be tested inside the test functions/methods, such as we (redundantly?) have in line 2933?

Reply:
I would say we "used" to do that, but I haven't found it necessary since using pytest. I believe the main argument for doing the imports inside the test is that `unittest` would fail completely if an import failed. I'm not sure pytest has that problem. Regardless I'm not sure I agree with this practice anymore as it produces a lot of redundant code with no benefit. If the import fails the tests fail in one way or another and I'm ok with that.